bin/i18n/test/translation_status/test_translation_service.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../../../shared/test/common_test_helper'
+require_relative '../../translation_status/translation_service'
+
+class TranslationServiceTest < Minitest::Test
+  # Use the same service for all the tests because initialization is very slow
+  @@translation_service = TranslationService.new
+
+  def test_translated_given_english_hello_should_return_true
+    assert @@translation_service.translated?('en-US', 'hello')
+  end
+
+  def test_translated_given_german_hello_should_return_true
+    assert @@translation_service.translated?('de-DE', 'hello')
+  end
+
+  def test_translated_given_es_hello_should_return_true
+    assert @@translation_service.translated?('es-ES', 'hello')
```

Comment:
one more question: is the locale case-sensitive? I've seen `'es-ES'`, `'es-es'` and `:"es-ES"` (symbol) in our code.

Reply:
I added a lower-case 'en-us` test and it passed.